Guisan.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure that `*.user` should be versioned.

Reply:
It shouldn't, thanks. Added to gitignore and removed the files from the repo.